### PR TITLE
Numerous detective related changes.

### PR DESCRIPTION
**Pull request by pireamaineach**
Cleans up the kit of detective plasmamen, the addition of the loose tie allowing them to easily remove the tie, preventing clipping issues with their envirosuit. The white helmet provides them with more versatility to wear clothing such as the noir coat, and the bowman headset was replaced to prevent clipping issues with the helmet.
(After removing the tie) New:
New
Old:
Old

New Noir Suit:
NoirSuit
Edited Noir Trenchcoat:
NoirTrenchcoat

Revolver Sprites:
Revolvers

:cl:
Adds the loose tie.
Adds the noir suit coat.
Noir suit coat now spawns in the detective's locker.
The detective spawns with a loose tie.
Removes the tie from the trenchcoat's sprites, the loose tie mirroring it exactly.
Plasmaman detectives spawn with a white helmet.
Plasmaman detectives now spawn with a normal security headset.
Resprites/Recolours the noir trench coat, making it darker to match the fedora.
Resprites the colt detective special to actually look like a colt detective special.
Adds two new reskin options for the detective special inspired by historical variants of the detective special, the police positive and fitz special, as well as blued and stainless steel variants.
Renames the "Mars .38 Special" to Colt Detective Special because there's no real reason why it shouldn't be the actual name.
/:cl:

### CODE REVIEW

**Comment by as334** [comment]
Nice job. This looks a lot nicer.

**Review by ShizCalev — DISMISSED** [review]

**Comment by pireamaineach** [comment]
Now it's actually ready; don't quote me. I should probably not commit every time I make a tiny change but fuck it.

**Comment by subject217** [comment]
pretty bikesheddy tbh, are you making these for your static character

**Comment by pireamaineach** [comment]
I main atmos tech and the noir suit changes were at the request of a detective main.

**Comment by subject217** [comment]
🤷‍♀️ 

i am drunk so i am probably crazy shouldtn comment on prs tbh

**Comment by pireamaineach** [comment]
I like that with this comment he kept the downvote.

**Comment by anconfuzedrock** [comment]
brb downvoting

**Comment by pireamaineach** [comment]
my dad works at github ill ban u kid

**Comment by tralezab** [comment]
You did the changelog wrong

Also please dont rename the detective gun, the last one was fine. Otherwise I like it I guess

**Comment by Arianya** [comment]
Have you checked whether the Colt Detective Special's name/brand is copyrighted? It might seem like a stupid question but I recall this coming up in the past elsewhere in relation to well known guns and the manufacturer's being copyright happy.

Also I'm not crazy about the new noir suit. is there any reason to replace the old noir suit and not make this a new outfit? (and probably give the Detective a clothing vendor, it may be overdue).

**Comment by pireamaineach** [comment]
> Have you checked whether the Colt Detective Special's name/brand is copyrighted? It might seem like a stupid question but I recall this coming up in the past elsewhere in relation to well known guns and the manufacturers being copyright happy.
> 
> Also I'm not crazy about the new noir suit. is there any reason to replace the old noir suit and not make this a new outfit? (and probably give the Detective a clothing vendor, it may be overdue).

Colt isn't copyright happy in the slightest and we've several times used the name of their other models, notably the m1911.
Clothing vendor seems unnecessary and the noir suit was by request, I don't know anyone who's particularly favourable to the current noir suit anyway.

**Comment by pireamaineach** [comment]
> You did the changelog wrong
> 
> Also please dont rename the detective gun, the last one was fine. Otherwise I like it I guess

The new name is just as fine, it's not a significant change.
It's my changelog I'll do what I want.

**Comment by Arianya** [comment]
I like the trenchcoat, and one person requesting a change isn't really justification for removal of an outfit and replacing it with another.

Not in a position to check it personally, but by the way you're talking about it, the waistcoat is also baked into into the new outfit instead of being optional? Thats a big :-1: from me considering we already have the optional waistcoat.

Also changing balance (bowman vs normal headset) purely for aesthetic is fairly :confused:. I doubt anyone will complain given the limited cross section of detective players and plasmaman players but worth noting.

**Comment by pireamaineach** [comment]
> I like the trenchcoat, and one person requesting a change isn't really justification for removal of an outfit and replacing it with another.
> 
> Not in a position to check it personally, but by the way you're talking about it, the waistcoat is also baked into into the new outfit instead of being optional? Thats a big  from me considering we already have the optional waistcoat.
> 
> Also changing balance (bowman vs normal headset) purely for aesthetic is fairly . I doubt anyone will complain given the limited cross section of detective players and plasmaman players but worth noting.

The waistcoat is still optional and the plasmaman envirohelmet protects the ears.

**Comment by ExcessiveUseOfCobblestone** [comment]
I actually prefer the old noir jacket :(

**Comment by pireamaineach** [comment]
> I actually prefer the old noir jacket :(

I'll see what I can do.

**Comment by PKPenguin321** [comment]
>revolver is one of the few items with a versatile custom sprites system so we can add new ones without replacing the old ones
>uhhhhh lets replace the old ones anyways lol

Dislike the new gun sprites, the contrast is way too low and it makes all the detail muddy-looking and mesh together, but you could still add them as alt skins
Also WHITE HELMET WTF

**Comment by pireamaineach** [comment]
> > revolver is one of the few items with a versatile custom sprites system so we can add new ones without replacing the old ones
> > uhhhhh lets replace the old ones anyways lol
> 
> Dislike the new gun sprites, the contrast is way too low and it makes all the detail muddy-looking and mesh together, but you could still add them as alt skins
> Also WHITE HELMET WTF

It's because you're looking at it on a white background.
Also I'm supposed to add an alternate sprite of the colt detective special that actually looks like a colt detective special, instead of just respriting it?
Tell you what though, I should add an alternate sprite that's silver as it orignally was.

**Comment by ExcessiveUseOfCobblestone** [comment]
The gun should be able to be renamed so it doesn't _really_ matter how 1:1 it is with the initial name of the gun.

**Comment by pireamaineach** [comment]
Yeah, I just don't see why it shouldn't be 1:1. Again, it's a minor change.

**Comment by ExcessiveUseOfCobblestone** [comment]
Because I want it to be the best bikeshed this game has to offer.

**Comment by pireamaineach** [comment]
Alright boys I've decided that to appease PKpenguin's being autistic I shall instead of just flat-out improving the visuals of the detective's gun I will leave in the out-dated sprites and add the new ones as alternate skins even though it's incredibly redundant.

**Comment by pireamaineach** [comment]
God bless.

**Comment by pireamaineach** [comment]
Unironically if you have a problem with the P.R. take it up with me and I'll see about finding a solution.

**Comment by ExcessiveUseOfCobblestone** [comment]
(* ´>艸<)゛

**Comment by PKPenguin321** [comment]
Thanks dude, careful with saying a****** here though or githubs robots will autoban you

**Comment by pireamaineach** [comment]
> Thanks dude, careful with saying a****** here though or githubs robots will autoban you

No problem I'm definitely doing this 100%.

**Comment by pireamaineach** [comment]
God damn it moo stop being silly.

**Comment by pireamaineach** [comment]
Does this please the masses?

**Comment by pireamaineach** [comment]
I swear to fucking god if I put on this ready tag and you all start complaining.

**Comment by Arianya** [comment]
Good to have the trenchcoat. Flared bottom looks a bit weird on github screenshot but might work fine in game?

**Comment by pireamaineach** [comment]
"I like the trenchcoat," says Arianya.
"The trenchcoat looks a bit weird," says Arianya.

**Comment by pireamaineach** [comment]
What the fuck why?

**Comment by pireamaineach** [comment]
wHy

**Comment by ExcessiveUseOfCobblestone** [comment]
press f

**Comment by pireamaineach** [comment]
This wouldn't even matter if it was merged it's nearly exactly the same what the fuck. Am I just going to have the merge conflict tag and a red x because of github hack chicanery?

**Comment by pireamaineach** [comment]
Mother fucker--

**Comment by pireamaineach** [comment]
This just isn't going to work, fuck it. Barely even a conflict, the new file even has the changes made.

**Comment by pireamaineach** [comment]
Even the sprite difference bot gave up on this P.R.

**Comment by pireamaineach** [comment]
Icon difference bot is back, everything seems fine enough besides me having wasted a lot of time.

**Comment by ATH1909** [comment]
Maybe the detective should get its own drop pod beacon for detective outfits, like what the chaplain gets for their holy armor?

Maybe you could make the kind of armor that you spawn with as a detective a part of your preferences, sort of like how sec officers can choose their starting department in their preferences?

**Comment by pireamaineach** [comment]
Seems very, very unnecessary considering the locker works just fine.

**Comment by pireamaineach** [comment]
What the fuck. It literally just doesn't let me update the file at all.

**Comment by pireamaineach** [comment]
english is my second language

File: code/modules/projectiles/projectile/bullets/revolver.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 /obj/item/projectile/bullet/c38
 	name = ".38 bullet"
-	damage = 25
+	damage = 30
```
Comment (PKPenguin321) [inline code/modules/projectiles/projectile/bullets/revolver.dm]: seems kind of like a stealth change

**Comment by pireamaineach** [comment]
Real' stealthy the way I asked people ahead of time and specifically noted it in the commit.

File: code/modules/projectiles/projectile/bullets/revolver.dm (hunk 1 above)
Comment (pireamaineach) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: Real' stealthy the way I asked people ahead of time and specifically noted it in the commit.

Comment (PKPenguin321) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: not in PR body or changelog or anywhere but the commit and in a PR that's primarily for cosmetic changes

Comment (pireamaineach) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: Sorry that I didn't update the changelog after all this shit. I just want this to be over with, the merge conflict was a headache.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/projectile/bullets/revolver.dm]: revert this and put it in a separate PR then i'll merge this PR

```suggestion
	damage = 25
```

**Pull request merged**